api/swagger.yaml

```diff
@@ -77,43 +90,337 @@ tags:
+      | ●        |         |          |       | ●     |       |            |        |       |
+
+
+      ## Action events
+
+      Action events describe actions that take place in an operation. Action events can typically be used in any life cycle phase. This makes sense,
+      since you can say an action can be planned, projected, actually happening or be realized. Action events typically involve only one entity at
+      a time.
+
+      #### Applicable life cycle phases
+
+      | Planned | Projected | Actual | Realized |
+      | :-----: | :-------: | :----: | :------: |
+      |  ●      | ●         | ●      | ●        |
+
+      ### `startMovingEvent`, `stopMovingEvent`
+
+      Marks the moment a `Vehicle` starts or stops moving after or before standing still for a while.
+
+      #### Applicable entities
+
+      | Location | Vehicle | Shipment | Trip  | Route | Actor | Constraint | Sensor | Event |
+      | :------: | :-----: | :------: | :---: | :---: | :---: | :--------: | :----: | :---: |
+      |          | ●       |          |       |       |       |            |        |       |
+
+      ### `startWaitingEvent`, `stopWaitingEvent`
+
+      Marks the moment a `Vehicle` starts or stops waiting.
+
+      #### Applicable entities
+
+      | Location | Vehicle | Shipment | Trip  | Route | Actor | Constraint | Sensor | Event |
+      | :------: | :-----: | :------: | :---: | :---: | :---: | :--------: | :----: | :---: |
+      |          | ●       |          |       |       |       |            |        |       |
+
+      ### `startLoadingAndUnloadingEvent`, `stopLoadingAndUnloadingEvent`
+
+      Marks the start or end of a time period meant for loading and unloading.
+
+      #### Applicable entities
+
+      | Location | Vehicle | Shipment | Trip  | Route | Actor | Constraint | Sensor | Event |
+      | :------: | :-----: | :------: | :---: | :---: | :---: | :--------: | :----: | :---: |
+      |          | ●       |          |       |       |       |            |        |       |
+
+      ## Time window events
+
+      Time window events are used to model a time window in a `planned` or `projected` life cycle phase. The start and end of a time window are
+      modelled as separate events. The following time window events are available:
+
+      * `pickupTimeWindowStartEvent`: Marks the start of a time window for the pickup of a `Shipment`. Only applicable in `planned` or `projected` life cycle phases.      `
+      * `pickupTimeWindowEndEvent`: Marks the end of a time window for the pickup of a `Shipment`. Only applicable in `planned` or `projected` life cycle phases.
+      * `deliveryTimeWindowStartEvent`: Marks the start of a time window for the delivery of a `Shipment`. Only applicable in `planned` or `projected` life cycle phases.
+      * `deliveryTimeWindowEndEvent`: Marks the end of a time window for the delivery of a `Shipment`. Only applicable in `planned` or `projected` life cycle phases.
```

Comment:
Since all time window events are only applicable in planned or projected, we could omit the statement in every line here.